### ISSUE TITLE
[Keyboard Navigation - Data Explorer (iframe)]]: ‘Expand and collapsed’ icon button is not accessible through keyboard, present in 'Editor Pane' under New Document.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1705]
Copied from Azure CosmosDB: https://msdata.visualstudio.com/CosmosDB/_workitems/edit/524363

URL: https://ms.portal.azure.com/

**User Experience:**
User rely on keyboard will find difficulty to interact with the controls  if  controls are not accessible through keyboard as user won't be able to perform functionality over such control.

**Test Environment:**
OS: Version 1903(18362.10022)
Browser: Edge, Chrome

**Pre-Requisite:**
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

**Repro Steps:**
1. Open URL: https://ms.portal.azure.com/
2. Login via valid credentials
3. Navigate to "Azure Cosmos DB " from left pane and select it.
4. Navigate to any of the Azure Cosmos DB which is already created and select it.
5. Navigate to "Data Explorer" from the left and select it.
6. Navigate through all the controls under SQL API Section and select Documents item.
7. Navigate to "New Document" button and select it.
8. Editor Pane will Open.
9. Navigate to  all the controls via keyboard and verify Expand and collapsed button is accessible or not.

**Actual Result:**
‘Expand and collapsed’ icon button is not accessible through keyboard, present in 'Editor Pane' under New Document.

**Expected Result:**
‘Expand and collapsed’ icon button should be accessible through keyboard, present in 'Editor Pane' under 'New Document'.

**Note:**
Similar issue exist with Chrome.
Similar issue exist on "Graph tab" under scenario: Azure Cosmos DB->Data Explorer->Graph->Load Graph

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b8492c4eb-c179-40ae-8777-cd044ed725a2%7d

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug.

For more information please use below link:
https://nam06.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx?from%3DSendByEmail%26e%3Dm-7ijwbaZECqFhJI4XGCQQ%26at%3D9&data=02%7c01%7cv-sucham%40microsoft.com%7cd795ba5e74de4e4b396e08d737711462%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637038831139684525&sdata=cig1QW25eorlCKQB5BXov6UIxg7OvCX8wOuG5ywHBnU%3D&reserved=0

**Attachment:**
[MAS2.1.1_editor.zip](https://github.com/sonimishra/Azure-CosmosDB-Accessibility-issues/files/3963768/MAS2.1.1_editor.zip)

**Comment by alexdima** [comment]
The expand/collapse buttons are not in the tab ring by design. Folding (expand/collapse) can be triggered via the keyboard e.g. by pressing `F1` and typing `fold`:

![image](https://user-images.githubusercontent.com/5047891/70893756-fe278580-1feb-11ea-9665-bbea3b159254.png)

There are even default keyboard shortcuts to do so e.g. `Ctrl+Shift+[`
